Makefile

```diff
@@ -118,8 +118,9 @@ pylava: $(PELORUS_VENV)
 # shellcheck follows a similar pattern, but is not currently set up for CI.
 
 chart-lint: $(PELORUS_VENV)
+	./scripts/install_dev_tools -v $(PELORUS_VENV) -c ct && \
 	. ${PELORUS_VENV}/bin/activate && \
-	./scripts/chart-lint
+	ct lint --config ct.yaml
```

Comment:
Not really. If no --upstream is specified then origin is used:
`--remote string                  The name of the Git remote used to identify changed charts (default "origin")`

So for forked repo origin is forked repo, for PR it will become github.com/konveyor/pelorus. I think we are fine here.